src/main/java/sungshin/sooon/controller/PostController.java

```diff
@@ -0,0 +1,78 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.domain.entity.Account;
+import sungshin.sooon.domain.entity.CurrentUser;
+import sungshin.sooon.dto.PostRequestDto;
+import sungshin.sooon.dto.PostResponseDto;
+import sungshin.sooon.service.PostService;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/posts")
+@Validated
+public class PostController {
+    private final PostService postService;
+
+    /*
+    //내가 쓴 글 조회
+    @GetMapping("/")
+    public ResponseEntity findAllByAccount(@CurrentUser Account account)
+    {
+        List<PostResponseDto> posts = postService.findAllByAccount(account);
+        return new ResponseEntity(posts,HttpStatus.OK);
+    }*/
+
+    @GetMapping("/")
```

Comment:
```suggestion
    @GetMapping("")
```

https://sanghaklee.tistory.com/57 

> 후행 슬래쉬(/)는 URI에 포함되지 않아야 합니다. 
계층관계를 나타낼 때 슬래시 구분자를 사용해야합니다.
출처: https://sas-study.tistory.com/265